src/test/kotlin/cacophony/TestUtils.kt

```diff
@@ -0,0 +1,69 @@
+package cacophony
+
+import cacophony.automata.DFA
+import cacophony.utils.Input
+import cacophony.utils.Location
+
+fun createDFA(
```

Comment:
There is simpleDFA that does exactly this. Maybe it's still better to mock it, but seems weird if code is almost the same.

Reply:
nice, I changed it to use simpleDFA. Also went ahead and did the same in `DFAEquivalenceTest.kt`